invokeai/app/cli/commands.py

```diff
@@ -47,7 +47,7 @@ def add_parsers(
     commands: list[type],
     command_field: str = "type",
     exclude_fields: list[str] = ["id", "type"],
-    add_arguments: Callable[[argparse.ArgumentParser], None]|None = None
+    add_arguments: Union[Callable[[argparse.ArgumentParser], None],None] = None
```

Comment:
```suggestion
    add_arguments: Optional[Callable[[argparse.ArgumentParser], None]] = None
```
Note: need import Optional from typing